frontend/templates/troubleshooting/Resource.tsx

```diff
@@ -1,20 +1,137 @@
-import { Stack, Image, Badge, StackProps, Link, Wrap } from '@chakra-ui/react';
+import {
+   Stack,
+   Image,
+   Badge,
+   StackProps,
+   Link,
+   Wrap,
+   HStack,
+   Text,
+   Box,
+   SystemProps,
+   Button,
+   ButtonProps,
+   ThemingProps,
+   useBreakpoint,
+} from '@chakra-ui/react';
 import { Guide } from './hooks/GuideModel';
 import { FaIcon } from '@ifixit/icons';
 import { faClock } from '@fortawesome/pro-solid-svg-icons';
 import Prerendered from './prerendered';
 import { DifficultyThemeLookup, GuideDifficultyNames } from './DifficultyBadge';
+import { Product } from '@models/product';
+import { useSelectedVariant } from '@templates/product/hooks/useSelectedVariant';
+import { useIsProductForSale } from '../product/hooks/useIsProductForSale';
+import { shouldShowProductRating } from '../../../packages/helpers/product-helpers';
+import { Rating } from '@components/ui';
+import { Money, formatMoney } from '@ifixit/helpers';
 
 export function GuideResource({ guide }: { guide: Guide }) {
    return (
       <Resource
          href={guide.url}
          title={guide.title}
          imageUrl={guide.image.thumbnail}
-         introduction={guide.introduction_rendered}
          timeRequired={guide.time_required}
          difficulty={guide.difficulty}
-      />
+         spacing="6px"
+      >
+         {guide.introduction_rendered && (
+            <Prerendered
+               lineHeight="1.36"
```

Comment:
Was this pulled from the design? `line-height: 136%;` Just curious.

Reply:
I think so. I didn't add this in this pull, I just moved it.